doc/release/1.9.0-notes.rst

```diff
@@ -26,42 +26,117 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 **************************
 
+- Added `scipy.optimize.milp`, new function for mixed-integer linear
+  programming.
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions
+  to data.
+- Added new spline based interpolation methods for
+  `scipy.interpolate.RegularGridInterpolator`.
+- A new global optimizer (DIviding RECTangles algorithm)
+  `scipy.optimize.direct`
+
 
 ************
 New features
 ************
 
+Lazy access to subpackages
+==========================
```

Comment:
I'd say that this doesn't belong here, it should go in the `Other changes` section, like the build system changes. `New features` is for subpackages only.